releases/nightly-build/compose-files/Makefile

```diff
@@ -66,5 +66,14 @@ clean: down
 	docker network prune -f
 
 get-token:
-	cd source; \
-	make get-token ${ARM64_OPTION}
\ No newline at end of file
+	docker run --rm \
+		-e SECRETSERVICE_SERVER=edgex-vault \
+		-e KONGURL_SERVER=kong \
+		-e SECRETSERVICE_TOKENPATH=/tmp/edgex/secrets/edgex-security-proxy-setup/secrets-token.json \
+		-e SECRETSERVICE_CACERTPATH=/tmp/edgex/secrets/ca/ca.pem \
+		--network edgex_edgex-network \
+		--volume /tmp/edgex/secrets/ca:/tmp/edgex/secrets/ca:ro,z \
+		--volume /tmp/edgex/secrets/edgex-security-proxy-setup:/tmp/edgex/secrets/edgex-security-proxy-setup:ro,z \
+		nexus3.edgexfoundry.org:10004/docker-edgex-security-proxy-setup-go$(ARM64):master \
```

Comment:
Should this image name be `docker-security-proxy-setup` instead of `docker-edgex-security-proxy-setup-go`?

Reply:
Agree with Jim.

Reply:
YES!!! Nice catch!!